### ISSUE TITLE
Wrong transformation on normals

### ISSUE THREAD

**Issue by JRK80** [OGRECave/ogre-assimp#6]
I noticed this when trying to convert some of our assets to .mesh and displaying it with a PBS gold texture. The reflection was completely off and the models looked almost flat. After logging the normal vectors, I could see that all normals pointed almost into the same direction.

In AssimpLoader::createSubMesh() you use the transformation matrix aiM on both the position and the normal vertices. This creates incorrect normal vertices when the transformation matrix has a translation component.

I changed the function to create a transformation matrix without the translation component and used the new matrix on the normals. This fixes my models.

Here is the code I used:

```
aiMatrix4x4 aiM = mNodeDerivedTransformByName.find(pNode->mName.data)->second;

aiQuaternion rotation;
aiVector3D position;
aiVector3D scaling;
aiM.Decompose(scaling, rotation, position);

aiMatrix4x4 scalingM;
aiMatrix4x4::Scaling(scaling, scalingM);

aiMatrix4x4 rotationM = aiMatrix4x4(rotation.GetMatrix());

aiMatrix4x4 normalMatrix = scalingM * rotationM;

.
.
.

// Normal
if (norm)
{
    vect.x = norm->x;
    vect.y = norm->y;
    vect.z = norm->z;

    vect *= normalMatrix;
    vect = vect.Normalize();

    *vdata++ = vect.x;
    *vdata++ = vect.y;
    *vdata++ = vect.z;
    norm++;

    //*vdata++ = norm->x;
    //*vdata++ = norm->y;
    //*vdata++ = norm->z;
    //norm++;
}
```

**Comment by paroj** [comment]
can you create a pull-request for this?

**Comment by JRK80** [comment]
I tried pushing to the repository, but always getting 
```
remote: Permission to OGRECave/ogre-assimp.git denied to JRK80.
fatal: unable to access 'https://github.com/OGRECave/ogre-assimp.git/': The requested URL returned error: 403
Pushing to https://github.com/OGRECave/ogre-assimp.git
```

I haven't worked with git or github before, so I have no clue if I'm doing something wrong or if I'm just missing some rights.
I think for a pull request I need to create a branch, but I can't do that either.

Edit: Ok, after some googling it seems I need to fork first... lets see

**Comment by paroj** [comment]
you need to fork the repository first and push your changes to the fork. After that you can do:
https://help.github.com/articles/creating-a-pull-request-from-a-fork/

**Comment by JRK80** [comment]
Ok, pull request is created. Was surprisingly easy, when you do it the right way ;)

**Comment by JRK80** [comment]
Fixed in 37ef7ec898ef9f4bbd1b1461d7ea131abb66df56
